src/main/java/org/ohdsi/analysis/Utils.java

```diff
@@ -74,4 +79,42 @@ public static String serialize(Object object) {
             throw new RuntimeException(ex);
         }
     }
+
+    public static <T> T deserialize(String data, JavaType typeRef) {
+        ObjectMapper objectMapper = getObjectMapper();
+        try {
+            return objectMapper.readValue(data, typeRef);
+        } catch (NullPointerException ex) {
```

Comment:
I'd rather do a check to see if returning null is appropriate vs. catching any NPE and thinking that it means the result is null.